demo/index.html

```diff
@@ -1,29 +1,39 @@
 <!doctype html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
-  <title>vaadin-radio-button Examples</title>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"
+      />
+    <title>vaadin-radio-button Examples</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
-  <link rel="import" href="../../polymer/polymer.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
-  <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
-  <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <link rel="import" href="../../polymer/polymer.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-component-demo.html">
+    <link rel="import" href="../../vaadin-demo-helpers/vaadin-demo-snippet.html">
+    <script src="../../vaadin-demo-helpers/vaadin-demo-ready-event-emitter.js"></script>
+    <script src="radio-button-demo.js"></script>
 
-  <script src="radio-button-demo.js"></script>
+    <link rel="import" href="../../iron-form/iron-form.html">
```

Comment:
You can remove the `iron-form` import as you no longer use it in demo.

Reply:
Done.